Source/CombatExtended/CombatExtended/MassBulkUtility.cs

```diff
@@ -24,9 +24,11 @@ public static float BaseCarryBulk(Pawn p)
         }
 
         public static float MoveSpeedFactor(float weight, float weightCapacity)
-        {
-            return Mathf.Lerp(1f, 0.75f, weight / weightCapacity);
-        }
+        {
+			float t = weight / weightCapacity;
+			if (float.IsNaN(t)) t = 0f;

```

Comment:
Should use min value instead of 0 to avoid AI derpiness.

Reply:
done